quintilesims/layer0
# 424: Modify L0 API ECS instance user data to send logs to CloudWatch

**What does this pull request do?**
The initial user data script that is executed at l0 api instance launch time will now stream 6 logs into their own log streams for the api instance's CloudWatch Log Group:
* /var/log/docker
* /var/log/ecs/cloudwatch-logs-start.log
* /var/log/ecs/ecs-agent.log
* /var/log/ecs/ecs-init.log
* /var/log/messages
* /var/log/user-data.log

**How should this be tested?**
Create a new layer0 api instance and observe the new log streams created for your l0 instance.

**Note:** You need to have the fixes from issue #455 and #456 applied to run this successfully.

~**Checklist**~
~- [ ] Unit tests~
~- [ ] Smoke tests (if applicable)~
~- [ ] System tests (if applicable)~
~- [ ] Documentation (if applicable)~

closes #424
